docs/versioning/versioning-overview.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+nav_order: 2
+title: Versioning Overview 
+has_children: false
+parent: Versioning
+---
+# Versioning Overview 
+
+Provena provides capability for versioning of a relevant subset of entity types in the Registry, including datasets. The purpose of this feature is to allow users to create and manage versions of registered items in Provena. Users can version items to mark items at a point in their development or release lifecycle. Users can then manage and refer to versions of an item. This can then provide a more accurate provenance trail, e.g. version 1 of Dataset X uses a different configuration of a modelling software suite to version 2 of Dataset X.
+
+Other use cases include:
+* Marking a previous version of an item as deprecated or erroneous
+* Maintaining two versions of items that are valid but have different lineages
+
+
+## What can be versioned?
+
+* Datasets
+* Model
+* Dataset template 
+* Model run workflow template
+
+## What cannot be versioned?
+
+* Agents - people, organisations
+* Model workflow run records 
+
+## What happens when an item is versioned
+
+1. A new persistent identifier is minted for each version (i.e. a new Handle ID given)
+2. A versioned item is timestamped
```

Comment:
Is this important?

Reply:
i think it can be useful... isn't it timestamped? (i think you wrote this in the confluence page originally too)